domains/mhdxr.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "mhdxr",
+           "email": "",
+           "discord": "529012608393674787",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.FKYkRwbjHc1pAo4fqIXjtJojBBjpfTYCR1ZqsUj2s1Y--Qi2yxVYsjELzxTQ-i-pzlKY6xFjJOUL1-2r65CiAUOGxlaIR5ksewb4vFFm7Tf8EFIO_c2sEq18j9uiZbbVdFw7gYT5gAFzjms5_JWZvy2-gWST4yCoFHhO7bjLXDlJW9W0QALtSxSMI7IK9GX_QO2Iik-QFA7TXMuTzz4XXtiJ5qzbaAR1GIhLX4CZNL9fZmDAMA2Jfiz8uC6vzREOxotDNV7mNwf4CKYozjoZnMXDyXNsyjUXTjDsEFTp_3AJLdyXF2zwyZkHfKpBC3tLCbyNgOQQwXpQA4tuehzzNA.Bgrx4aAjOKlxFAwDo5567A.Ul8LCFh2uHXNAgfZREjzYl0t5e2K8wHF1YY9sw3TSldY5mu8QidPoxB0PZgLfKf92qKTNvc7qH8Jojs97fMh1g.5w64qd_3Q7-YlPol_YQdqg"
+        },
+    
+        "record": {
+            "CNAME": "contact@mhdxr.tech"
```

Comment:
CNAME records are meant to point towards domains, not email addresses. 
Please read the [documentation](https://www.is-a.dev/docs/).